xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/dto/account/BitstampEarnSettingRequest.java

```diff
@@ -0,0 +1,32 @@
+package org.knowm.xchange.bitstamp.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+@Value
+public class BitstampEarnSettingRequest {
+
+  @JsonProperty("setting")
+  Setting setting;
+
+  @JsonProperty("currency")
+  String currency;
+
+  @JsonProperty("earn_type")
+  BitstampEarnType earnType;
+
+  public enum Setting {
+    OPT_IN,
+    OPT_OUT;
+
+    @JsonCreator
```

Comment:
is not needed

Reply:
Removed the unnecessary @JsonCreator annotation and the fromString method.